README.md

```diff
@@ -45,27 +45,29 @@ Or install it yourself as:
 Start with a domain model of POROs and AR::Base objects that form an aggregate:
 
 ```ruby
-class Tree; end
-
 class Branch
-  include Virtus.model
+  attr_accessor :id
+  attr_accessor :length
+  attr_accessor :diameter
+  attr_accessor :tree
 
-  attribute :id, Integer
-  attribute :length, Decimal
-  attribute :diameter, Decimal
-  attribute :tree, Tree
+  def initialize(id: nil, length: 0, diameter: nil, tree: nil)
+    @id, @length, @diameter, @tree = id, length, diameter, tree
+  end
```

Comment:
The Readme never shows how to instantiate a Branch, so this constructor can be left out.

Reply:
Vorpal requires this initializer. If I remove it from here, ppl won't be able to just copy and paste this code to try vorpal with the Readme example. Is that ok?

Reply:
Vorpal relies on getters and setters to serialize and deserialize, not initializers, so these constructors shouldn't be needed. But, I think you have found a bug in Vorpal when a has many association is set to `nil` rather than an empty array! I will fix this bug once you have removed the constructors.

Reply:
Done